src/components/modules/card/index.tsx

```diff
@@ -1,5 +1,19 @@
 import React from "react";
+import { ProductType } from "../../../services/types";
 
-export const Card = () => {
-  return <div>Card</div>;
+export const Card = (product: ProductType) => {
+  return (
+    <div className="product-card">
+      <img src={product.image} alt={product.name}></img>
```

Comment:
Nesse caso o alt tem uma informação redundante, pois já tem o nome da roupa abaixo. Quando a imagem é meramente ilustrativa e "sem valor" o ideal é deixar um alt com string vazia.

https://www.w3.org/WAI/tutorials/images/decorative/

```suggestion
      <img src={product.image} alt=""></img>
```